go.mod

```diff
@@ -189,6 +189,6 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.11.4 => github.com/ethereum-optimism/op-geth v1.11.2-de8c5df46.0.20230322010758-876919e42d47
+replace github.com/ethereum/go-ethereum v1.11.5 => github.com/ethereum-optimism/op-geth v1.11.2-de8c5df46.0.20230324105532-555b76f39878
```

Comment:
Convention that we bump the `go-ethereum` require to the latest version so we also have to bump the replace.

Reply:
From my understanding:
* The version we use when including `github.com/ethereum/go-ethereum` is arbitrary because we're replacing with a reference to a specific op-geth commit hash anyway - as Josh said, we tend to keep it inline with the upstream geth version.
* The version used in the replacement is automatically calculated by go - it looks for the latest tag on the `optimism` branch. We aren't pulling the upstream geth release tags across so that's not currently updating.  We could bring them over (they'd point to the actual commit upstream geth released - same as in the upstream repo) but I suspect that may cause some confusion.  There's probably some tagging improvements for the op-geth repo we could do.